src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -405,7 +419,7 @@ AmendmentTableImpl::AmendmentTableImpl(
             if (*vote_to_veto)
             {
                 // Unknown amendments are effectively vetoed already
-                if (auto s = get(amend_hash, sl))
+                if (auto s = get(amend_hash, lock))
                 {
                     JLOG(j_.info()) << "Amendment {" << *amendment_name << ", "
                                     << amend_hash << "} is vetoed.";
```

Comment:
Fixed, but I want to ask if you think this will cause confusion with the `feature` RPC command, which also uses `vetoed`: https://github.com/ximinez/rippled/blob/internalveto2/src/ripple/rpc/handlers/Feature1.cpp#L68

Reply:
That's a good question.  My feeling is that "vetoed" is an unfortunate left over from a long time ago when there were only five validators.  A single downvote really was pretty close to a veto.  With 41 validators it takes 9 downvotes to block an amendment.  So "downvoted" is more representative of the current state of things than "vetoed".

But I also think that it's not worth changing the API to clear this up. The costs of an API change are very high and need to be justified.  Users can live with "vetoed" in the API, even though the meaning is not precisely in line with the reality.

Given that, I still think it's good for our internal names to be clear that no one really gets "veto" power.

That's just my opinion.

Reply:
Yeah, that works for me.